manage_breast_screening/notifications/tests/management/commands/test_create_appointments.py

```diff
@@ -320,3 +330,22 @@ def test_calls_insights_logger_if_exception_raised(
                 Command().handle(**{"date_str": "Noooo!"})
 
         mock_insights_logger.assert_called_with("CreateAppointmentsError: Error!")
+
+    def test_create_extract_and_cancel(self):
+        """Test Extract creation for new booked appointments in NBSS data, stored in Azure storage blob"""
+        today_dirname = datetime.now().strftime("%Y-%m-%d")
+
+        with stored_blob_data(today_dirname, [SINGLE_APPOINTMENT_FILE]):
```

Comment:
could the other fields on an Extract be tested in this test? so sequence_number, bso_code, filename- would help validate the right things have been added